### ISSUE TITLE
[Bug - Android] DateTimeContiniousAxis binds different hour from the passed one

### ISSUE THREAD

**Issue by NickIliev** [telerik/nativescript-ui-feedback#33]
Using RadChart with DateTimeContiniousAxis and passing data via bind-model in form of hours the data is rendered with hour or two behind the actual data passed through the view-model.

Code to reproduce (using [https://github.com/telerik/nativescript-ui-samples](https://github.com/telerik/nativescript-ui-samples) )

Change the passed data model:
```

    get dateTimeSource() {
        return [
            { timeStamp: new Date(2015, 2, 2, 0), Amount: 1 },
            { timeStamp: new Date(2015, 2, 2, 1), Amount: 2 },
            { timeStamp: new Date(2015, 2, 2, 2), Amount: 3 },
            { timeStamp: new Date(2015, 2, 2, 3), Amount: 4 },
            { timeStamp: new Date(2015, 2, 2, 4), Amount: 5 },
            { timeStamp: new Date(2015, 2, 2, 5), Amount: 6 },
            { timeStamp: new Date(2015, 2, 2, 6), Amount: 7 },
            { timeStamp: new Date(2015, 2, 2, 7), Amount: 8 },
            { timeStamp: new Date(2015, 2, 2, 8), Amount: 9 },
            { timeStamp: new Date(2015, 2, 2, 9), Amount: 10 },
        ]
    }
```

And also modify chart >> axes >> date-time-axes.xml to:
```
<navigation:ExamplePage xmlns:navigation="navigation/example-page" loaded="onPageLoaded" xmlns:chart="nativescript-telerik-ui-pro/chart" xmlns="http://www.nativescript.org/tns.xsd">
    <chart:RadCartesianChart id="cartesianChart">
       <chart:RadCartesianChart.series>
                <chart:LineSeries items="{{ dateTimeSource }}" categoryProperty="timeStamp" valueProperty="Amount">
                    <chart:LineSeries.horizontalAxis>
                        <chart:DateTimeContinuousAxis majorStep="Hour" dateFormat="HH" />
                    </chart:LineSeries.horizontalAxis>
                    <chart:LineSeries.verticalAxis>
                        <chart:LinearAxis/>
                    </chart:LineSeries.verticalAxis>
                </chart:LineSeries>
            </chart:RadCartesianChart.series>
    </chart:RadCartesianChart>
</navigation:ExamplePage>
```

**Comment by VladimirAmiorkov** [comment]
Hi,

After researching this issue it looks like there is an unknow issue with the JavaScript's [JSON.stringify()](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify) which leads to incorrect date passed to the native RadChart elements. The Telerik UI library calls this method which introduces the described behavior.

In order to resolve the mentioned behavior all that you need is to pass the "date" object not as Date to from your model but as a Number (the date in milliseconds) by simply calling the `.getTime()` method of the Date object:

```
get dateTimeSource() {
        return [
             { timeStamp: new Date(2015,1,11).getTime(), Amount: 10}, 
            { timeStamp: new Date(2015,2,11).getTime(), Amount: 10}, 
            { timeStamp: new Date(2015,3,1).getTime(), Amount: 1 },
            { timeStamp: new Date(2015,4,3).getTime(), Amount: 3 },
            { timeStamp: new Date(2015,5,11).getTime(), Amount: 18}, 
            { timeStamp: new Date(2015,6,1).getTime(), Amount: 7 },
            { timeStamp: new Date(2015,7,3).getTime(), Amount: 5 },
            { timeStamp: new Date(2015,8,11).getTime(), Amount: 4}, 
            { timeStamp: new Date(2015,9,1).getTime(), Amount: 2 },
            { timeStamp: new Date(2015,10,3).getTime(), Amount: 6 },
           
        ]
}
```

**Comment by NickIliev** [comment]
Reopen as this is reproduced in t.1106204
This issue is likely to be related to the Summer Savings Time as it is happening when using 
new Date() on PC with EEST (EE Summer Time).

Code to reproduce needed in [this example](https://github.com/telerik/nativescript-ui-samples/tree/release/sdk/app/chart/axes/date-time-axes)

```
get dateTimeSource() {

    var now = new Date(); 
    console.log(now)
    var now1 = new Date();
    now1.setHours(now1.getHours() + 1);
    var now2 = new Date();
    now2.setHours(now2.getHours() + 2)
    var now3 = new Date();
    now3.setHours(now3.getHours() + 3)
    var now4 = new Date();
    now4.setHours(now4.getHours() + 4)
    console.log(">>>> now " + now);
    console.log(">>>> getTime " + now.getTime());
    console.log(">>>> converted " + new Date(now.getTime()));
    return [
        { timeStamp: now.getTime(), Amount: 10}, 
    ]
}
```

and the this in the XML file
```
<chart:LineSeries items="{{ dateTimeSource }}" categoryProperty="timeStamp" valueProperty="Amount">
    <chart:LineSeries.horizontalAxis>
        <chart:DateTimeContinuousAxis majorStep="Hour" dateFormat="HH-mm" labelFitMode="Rotate" labelRotationAngle="1.2"/>
    </chart:LineSeries.horizontalAxis>
    <chart:LineSeries.verticalAxis>
        <chart:LinearAxis/>
    </chart:LineSeries.verticalAxis>
</chart:LineSeries>
```

The rendered hour is not the same as the local time (hours + or - depending on the time zone from which the app is tested).

**Comment by ginev** [comment]
@NickIliev this issue will be addressed in the upcoming 2.0 release of the product.
